src/sentry/rules/history/preview.py

```diff
@@ -53,14 +58,13 @@ def preview(
     """
     Returns groups that would have triggered the given conditions and filters in the past 2 weeks
     """
-    # must have at least one condition to filter activity
-    if not conditions:
-        return None
-
     issue_state_conditions, frequency_conditions = categorize_conditions(conditions)
```

Comment:
just to confirm, `conditions` can never be `None`, right? I know it's disallowed by typing

Reply:
yup it should be a list

Reply:
nice just checking